docs/development.md

```diff
@@ -56,6 +63,10 @@ Make sure below settings are in root level of `.vscode/settings.json`.
 }
 ```
 
+While the black formatter does not handle long comments and the like
```

Comment:
How about add the explanation in "Other setups"? The link of rewrap can be provided like "Markdown All in One". It doesn't need to mention the black formatter, it's not related to the instruction.

Reply:
I'll keep the black remark. It's something it should be doing, instead of letting others do it manually.